### PR TITLE
fix block results upload url with ipfs prefix

### PR DESCRIPTION
**Pull request by sudeepdino008**

### CODE REVIEW

File: .envrc [hunk 1]
```diff
@@ -1,4 +1,3 @@
-export ERIGON_NODE="https://ethereum.web3.covalenthq.com/mainnet/rpc"
```
Comment (noslav) [inline .envrc:1]: good call removing this

File: lib/rudder/block_result/block_result_uploader.ex [hunk 2]
```diff
@@ -51,7 +51,7 @@ defmodule Rudder.BlockResultUploader do
                block_height,
                specimen_hash_bytes32,
                block_result_hash,
-               cid
+               "ipfs://#{cid}"
```
Comment (noslav) [inline lib/rudder/block_result/block_result_uploader.ex:54]: Looks like calling this fn with just the prefix addition should be enough!

**Review by noslav — APPROVED** [review]
LGTM 👍🏼

**Review by rogarcia — APPROVED** [review]
LGTM

**Pull request merged**